### DISCUSSION TITLE
Punkte/Symbole. Beschriftungen gehen verloren

### DISCUSSION THREAD

**Discussion by thomaskofler** [e-netze/webgis-community#98] [General]
Wenn man einen Punkt oder ein Symbol zeichnet und beschriftet, dann die Karte speichert und neu lädt, gehen die Beschriftungen der Punkte/Symbole verloren. Bei Linien und Flächen funktioniert es, da bleibt die Beschriftung.

**Comment by gis-eni** [comment]
Scheint das gleiche Problem gewesen zu sein wie bei https://github.com/gis-eni/webgis-forum/discussions/100

Mit Build 5.23.3601 behoben
